### DISCUSSION TITLE
API Platform & Mercure, how to handle POST ?

### DISCUSSION THREAD

**Discussion by vince83110** [api-platform/api-platform#2121] [Q&A]
We use API Platform & Mercure bundle, it works very well when you are updating an entity.

But, we want (for example), to automaticaly add a new entity in our listing when someone has created a new one.

I can't see in documentation how to handle it? Because topic like `/topic/products/{product}/items` will work only for existing products.

For now, I use DataPersister on POST event, and I will write manually on the topic I have created (with CQRS way, Message + 
MessageHandler, like `ProductHasBeenCreatedMessage`)

I have created a topic I call (for example): `/topic/product/append/items`, but I can't do it for all my entities, what is the good way to handle POST of new entities ? Did I miss something in doc ?

**Comment by langziyang** [comment]
i find the answer : 
```
useEffect(() => {
        const eventSource = new EventSource('http://192.168.100.5:3000/.well-known/mercure?topic=http://192.168.100.5/api/employees/{id}')
        eventSource.onmessage = event => {
            // Will be called every time an update is published by the server
            console.log(JSON.parse(event.data));
        }
        return () => {
            eventSource.close()
        }
    }, []);
```
do not change {id} to any other char
